ui/spells/spells.html

```diff
@@ -93,7 +93,22 @@
       <input name="attr_spellrange" class="spellrange-grid" type="text" />
       <input name="attr_spelltarget" class="spelltarget-grid" type="text" />
       <input name="attr_spellduration" class="spellduration-grid" type="text" />
-      <input name="attr_spellhasnote" class="spellhasnote-grid" type="checkbox" value="1" />
+      <div>
+        <button name="act_togglepopoverspellnote" class="popoverButton smallerPopoverButton" type="action">
+          ?
+        </button>
+        <!-- The next input puts whether we have a note into the
+                  value property, where CSS can see it -->
+        <input name="attr_visiblepopover" class="popoverController" type="hidden" value="false" />
+        <div class="spellNotePopover popoverContainer">
+          <div class="popover">
+            <!-- Buttons in a popover break stacking order, and pop out of the repeating stack. -->
+            <button name="act_closepopoverspellnotes" class="closePopoverButton" type="action">X</button>
```

Comment:
Turns out that `position: relative` breaks Roll20 and the button's action functionality.
This button can only act as a top level button, which is fine... We want to close all popovers when we click on this.